File: HA2_4.c
//Accept number from user and check wheather it is even or odd

#include<stdio.h>
#include<stdbool.h>
#define TRUE 1
#define FALSE 0

bool ChkNo(int iNo)
{
    if((iNo %2)==0)
    {
        return TRUE;
    }
    else
    {
        return FALSE;
    }
}

int main()
{
    int iValue = 0;
    bool bRet = FALSE;

    printf("Enter a number \n");
    scanf("%d",&iValue);

    bRet = ChkNo(iValue);

    if(bRet == TRUE)
    {
        printf("EVEN");
    }
    else
    {
        printf("ODD");
    }

    return 0;
}

File: HA2_4.py
#Accept number from user and check wheather it is even or odd

def ChkNo(iNo):
    if((iNo % 2)==0):
        return True
    else:
        return False

def main():
    iValue = 0
    bRet = 0

    print("Enter a number")
    iValue = int(input())

    bRet = ChkNo(iValue)

    if(bRet == True):
        print("EVEN")
    else:
        print("ODD")

if __name__ == "__main__":
    main()